pypeit/biasframe.py

```diff
@@ -50,6 +52,33 @@ class BiasFrame(calibrationimage.CalibrationImage, masterframe.MasterFrame):
     frametype = 'bias'
     master_type = 'Bias'
 
+    @classmethod
+    def from_master_file(cls, master_file, par=None):
+        """
+        Instantiate from a master file
+
+        Args:
+            master_file (str):
+            par (:class:`pypeit.par.pypeitpar.FrameGroupPar`, optional):
+
+        Returns:
+            biasframe.BiasFrame:
+                The PypeItImage is loaded into self.pypeitImage
+
+        """
+        # Spectrograph
+        spectrograph, extras = masterframe.items_from_master_file(master_file)
```

Comment:
Let me add that when I'm developing, I find the number of lines of code required to read in a master file super annoying. For that reason, I think having utility functions that process the master file output into the required products in one line of code is very valuable. Whether these should be class methods, or just methods in the same module is not totally clear to me, and probably does not matter.

Reply:
It looks like we have 3 overlapping and differing opinions on this one.
I agreed the extras list in `items_from_master_file` could get unruly but 
as already have more than one way of saving master info (FITS, JSON)
I find it somewhat unavoidable.    If we eventually go to only FITS (possible)
this could be stripped down.

As for not opening the file more than once, I see that as good practice
but I don't feel it is worth the extra coding (e.g. it would include adding a new
method on `PypeItImage`.  

Bottom line -- I didn't change much on this one other than adding to the doc string.